config/initializers/friendly_id.rb

```diff
@@ -1,6 +1,12 @@
 FriendlyId.defaults do |config|
-  config.base = :name
-  config.use :slugged, :finders, FriendlyId::SequentialSlugs
+  module CustomNormalisation
+    def normalize_friendly_id(input)
+      # Use the babosa gem for better string parameterisation
+      super input.to_s.to_slug.truncate(150).normalize.to_s
+    end
+  end
 
+  config.base = :name
+  config.use :sequentially_slugged, :finders, CustomNormalisation
```

Comment:
I just pushed a branch with the finders add-on removed and it looks like there are still a handful of calls to the old magic finders: https://ci-new.alphagov.co.uk/job/govuk_whitehall_branches/6237/console